packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -278,6 +278,20 @@ const ProductDetail = ({category, product, isLoading}) => {
 
                 {/* Product Recommendations */}
                 <Stack spacing={16}>
+                    {isProductASet ? null : (
```

Comment:
Minor: can be simplified to this
```suggestion
                    {!isProductASet && (
```